tests/TestContext.cs

```diff
@@ -113,13 +131,13 @@ public async Task PushFilesOnDefaultBranch()
 
         await ExecuteCommand(outputHelper, "git", ["-C", this._repoPath, "add", "."]);
         await ExecuteCommand(outputHelper, "git", ["-C", this._repoPath, "-c", "user.email=idp@workleap.com", "-c", "user.name=IDP ScaffoldIt", "commit", "--message", "IDP ScaffoldIt automated test"]);
-        await ExecuteCommand(outputHelper, "git", ["-C", this._repoPath, "push", gitUrl, DefaultBranchName + ":" + DefaultBranchName, "--force"]);
+        await ExecuteCommand(outputHelper, "git", ["-C", this._repoPath, "push", gitUrl, $"{targetBranchName}:{targetBranchName}", "--force"]);
```

Comment:
Do we need `--force` as you now use a unique branch name?

Reply:
No longer needed!